src/remove_todo.js

```diff
@@ -0,0 +1,5 @@
+export default function removeTheList(myv, todos) {
+  todos.splice(myv, 1);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  window.location.reload();
```

Comment:
maybe its possible find another solution for not reload the page, this goes against user experience.